backend/db/mongoose.js

```diff
@@ -0,0 +1,7 @@
+const mongoose=require('mongoose')
+mongoose.connect('mongodb://127.0.0.1:27017/HashIDE', {
```

Comment:
Also we will need to provide a seperate MongoDb database for the same and the User credentials should be globally stored and not locally. Try using ur MongoDb cluster 
```suggestion
mongoose.connect(process.env.MongoDbKey, {
```
Add MongoDbKey in an env file and add it to .gitignore so that other peeps are not able to see that